deployment/network-operator/values.yaml

```diff
@@ -229,6 +234,10 @@ rdmaSharedDevicePlugin:
   # it must be provided by the user when deploying the chart
   # each entry in the resources element will create a resource with the provided <name> and list of devices
   # example:
+  containerResources:
+    - name: "rdma-shared-dp"
```

Comment:
why do we want to provide empty default value ?
why not comment it out ?

empty may actually mean: i want empty req and limits and let container-runtime do whatever it wants